src/getCoinCombination.test.js

```diff
@@ -7,4 +7,34 @@ describe('getCoinCombination', () => {
     expect(getCoinCombination)
       .toBeInstanceOf(Function);
   });
+  
+  test('should return 0`s if original data is 0', () => {
+    expect(getCoinCombination(0)).toEqual([0, 0, 0, 0])
+   });
+
+   test('should return [0, 0, 0, 4] if original data is 100', () => {
+    expect(getCoinCombination(100)).toEqual([0, 0, 0, 4])
+   });
+
+   test('should return [1, 1, 1, 1] if original data is 41', () => {
+    expect(getCoinCombination(41)).toEqual([1, 1, 1, 1]);
+   });
+
+   test('should return [0, 1, 1, 1] if original data is 40', () => {
+    expect(getCoinCombination(40)).toEqual([0, 1, 1, 1]);
+   });
+
+   test('should return [4, 0, 0, 0] if original data is 4', () => {
+    expect(getCoinCombination(4)).toEqual([4, 0, 0, 0]);
+   });
+
+   test('should return [0, 1, 0, 1] if original data is 30', () => {
```

Comment:
The test case is not descriptive enough. It should include more information about what the function is doing.